### ISSUE TITLE
Feature suggestion "system messages popup" particle

### ISSUE THREAD

**Issue by marktaylor46** [gantry/gantry5#1652]
This came up in forum a couple of times and I think it would be good if we could implement this in Gantry 5...

Rather than having just the "system message" particle is would be good to have an alternate particle "system messages popup" particle that would display a modal view of the system messages in an overlaid popup that you could then close once the system messages had been read. The particle should not show anything on the page until there are actually some system messages to show.

This would also help with the time when system messages content can push layouts around (especially in fixed header scenarios).

Just an idea.

**Comment by mahagr** [comment]
There's really no need to have another particle for this. You can do the same just by some CSS and JS. So its really a theme thing.

**Comment by marktaylor46** [comment]
It's an often requested item though so I thought we could add it in at some point. A lot of members are not capable of writing the necessary JS and hooks to do this - I must admit I can't think how to do it either.

**Comment by OptikWebCreative** [comment]
+1 for all platforms :)

**Comment by hacki65** [comment]
I really like the idea. 👍

**Comment by JVKeller** [comment]
Any movement on this? 👍

**Comment by N8Solutions** [comment]
Thank you @marktaylor46 !

**Comment by mariantanase** [comment]
Hi Mark,
Where can I find the atom ? I don't see it

**Comment by marktaylor46** [comment]
On the link I provided above. It’s attached to the post.

**Comment by mariantanase** [comment]
Is visible only for rockettheme members I supposed. I cannot see it as a guest.

**Comment by marktaylor46** [comment]
Yes, you will need an RT account to be able to see that forum but you don't have to have any current subscriptions...

**Comment by mariantanase** [comment]
Sorry but there is no free RT account, I don't see any register form for registration a free account.
"_If you do not have an account, Join a RocketTheme Club or purchase a product for access._"

**Comment by marktaylor46** [comment]
You need to have purchased an account at some time in the past (even if the subscriptions have expired). Sorry but there's nothing I can do abou this.

**Comment by mariantanase** [comment]
Unfortunately, I've never had a subscription on RT.
Thank you anyway for intention.

**Comment by thexmanxyz** [comment]
@marktaylor46 @mariantanase @mahagr @N8Solutions 
What about providing the Atom publicly? I would also take my time to create a GH repository for it together with the changes I already made on the popupmessage Atom? So other people are able to participate provide input and I also would guarantee that I add functionality to the Atom following the release! There is no magic implemented so I see no reason for hiding it behind a pay wall! What do you think?

**Comment by thexmanxyz** [comment]
@mariantanase @mahagr @marktaylor46 @N8Solutions @JVKeller @hacki65 @OptikWebCreative 

I finally created a full rework of the original Atom posted on [RT Forums](https://rockettheme.com/forum/gantry5-for-joomla/278683-popupmessages-atom-freebie-by-mrt?limitstart=0). In the [project repository](https://github.com/thexmanxyz/Popup-System-Messages-Gantry) you can find the install explanations as well as the features added. Jump to the lastest release [Download](https://github.com/thexmanxyz/Popup-System-Messages-Gantry/releases). It also contains some fixes as well as support for the Joomla Plugin System.
